packages/@uppy/companion/src/standalone/index.js

```diff
@@ -204,5 +204,4 @@ function server (inputCompanionOptions = {}) {
   return { app, companionOptions }
 }
 
-const { app, companionOptions } = server()
-module.exports = { app, companionOptions, server }
+module.exports = { server }
```

Comment:
Maybe change it to `module.exports = server` ? Becase there's nothing else to export here

Reply:
makes sense - I changed it, thanks for the help!